avaframe/com1DFA/varyRelTh.cint

```diff
@@ -135,6 +136,9 @@ void RunExport(char BasePath[], char SamosProjectDir[], char ResDir[], int count
       Mu = 0.155;
       relHST = 1.0;
       printf("\n[BatchSamos] Computing %s with standard Mu=%f and standard RelTh=%f\n",CuSim,Mu,relHST);
+      sprintf(RelAr,"%s/ReleaseAreas",Sim);
+      sprintf(relHSTString,"%.3f",relHST);
+      set(RelAr, "set_depth", relHSTString);
```

Comment:
what does relHST stand for?
release hight standard?
and why then relTh in the description?
-> I guess the next important issue is the glossary.. :smile:

Reply:
relHST is the standard one and relTh has to be specified in the .ini file if you want variations of the release depth

Reply:
yes exactely as the null simulations follow the entres where relH has been set according to inputs the standard relH is imposed with relHST